upstream-community-operators/sysdig/sysdig-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sysdig-operator.v1.4.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "Security, Monitoring"
+    certified: "false"
+    description: |-
+      Sysdig is a unified platform for container and microservices monitoring, troubleshooting, security and forensics. Sysdig platform has been built on top of Sysdig tool and Sysdig Inspect open-source technologies.
+    containerImage: docker.io/sysdiglabs/sysdig-operator:1.4.0
+    createdAt: 2019-03-08T18:30:00Z
+    support: Sysdig, Inc.
```

Comment:
If there is a public git repository for this operator's source code, please add it under `metadata.annotations.repository`